src/main/webapp/api.js

```diff
@@ -0,0 +1,16 @@
+// Create the script tag, set the appropriate attributes
+var script = document.createElement('script');
+
+// Edit key between 'key=' and '&locat...' to our new API key once you pull this file.
+// Then add this file to .gitignore
+script.src = 'https://maps.googleapis.com/maps/api/js?key=AIzaSyB-kjcAxxbrQeJJEvSg5XgkTmGI4aZLS5E&libraries=places';
```

Comment:
API key still present in this file, are we hiding this entire file?

Reply:
Yes this is the file that we will each use to store the API key - once you guys pull this we are going to delete it from github, and then we are going to remove this old api key with a new one that we each individually place in this file, as well as use .gitignore to make sure we don't re-commit this new api key.

Reply:
Great solution! Are we sure this doesn't end up populating as an HTML element?